lib/overcommit/hook/pre_commit/es_lint.rb

```diff
@@ -8,9 +8,22 @@ module Overcommit::Hook::PreCommit
   # somewhere else. Example:
   #
   #   EsLint:
+  #     required_executable: 'npm'
   #     enabled: true
   #     command: ['npm', 'run', 'lint']
   #
+  # And currently this hook supports only eslint compact format, you can configure
+  # this in package.json. Example:
```

Comment:
I like the idea about `.overcommit.yml` to define the args.

But if you'd like to call `eslint` through `npm run`,you should define it in `package.json` like below:

```json
   {
     "scripts": {
       "lint": "eslint"
     },
     "devDependencies": {
       "eslint": "^2.0.0"
     }
   {
```